src/Scss/base/_responsive.scss

```diff
@@ -284,6 +289,37 @@ $largeDesktop: rem(1920);
   .user_reply_comment {
     padding-left: 0 !important;
   }
+
+  // Ticket Page
+  .tickets_details {
+    .table_footer {
+      justify-content: center !important;
+      align-items: center !important;
+    }
+  }
+
+  .tickets_search {
+    flex-wrap: wrap;
+    justify-content: center !important;
+    align-items: center !important;
+    padding: rem(30) !important;
+    height: auto !important;
+  }
+  .left_items {
+    flex-grow: 1;
+    input {
+      width: 100% !important;
+    }
+  }
+  .table_footer {
+    justify-content: center !important;
```

Comment:
Even here, you should have used flex mixin for this and whats the need of important in every property.

Reply:
Inside this style (tickets.component.scss), I have already applied `display: flex` and `justify-content: flex-start`, so I used `!important` to override the existing `display: flex` that was already applied. Therefore, I did not use `flex` here.